scripts/start_all.js

```diff
@@ -138,7 +164,8 @@ const startAll = async () => {
     process.exit(1);
   }
 
-  console.info(chalk.greenBright('Stack started successfully.'));
+  console.log(); // New line
```

Comment:
"\n" adding this to string should be enough for creating new line, but its not important change :D

Reply:
I know, I just never do it :P